keps/sig-cloud-provider/2436-controller-manager-leader-migration/README.md

```diff
@@ -178,16 +178,17 @@ type ControllerLeaderConfiguration struct {
 
 	// Component is the name of the component in which the controller will be running.
 	// E.g. kube-controller-manager, cloud-controller-manager, etc
+	// Or '*' meaning the controller can be run under any component that reads the configuration
 	Component string `json:"component"`
 }
 
 ```
 
 #### Default LeaderMigrationConfiguration
 
-The `staging/controller-manager` package will provide `kube-controller-manager` and `cloud-controller-manager`
-each a default `LeaderMigrationConfiguration` that represents the situation where the controller manager is running with
-default assignments of controllers and lock type selection.
+The `staging/controller-manager` package will provide a default `LeaderMigrationConfiguration` which has the
+`component` filed of all controllers that need migrating set to `*`, indicating the controllers can be run under
```

Comment:
Can we list the 4 controllers it will have this for. Might be good to just spell out the full default. 
Eg.
kind: LeaderMigrationConfiguration
apiVersion: controllermanager.config.k8s.io/v1alpha1
leaderName: cloud-provider-extraction-migration
resourceLock: leases
controllerLeaders:
  - name: route-controller
    component: *
  - name: service-controller
    component: *
  - name: cloud-node-controller
    component: *
  - name: cloud-nodelifecycle-controller
    component: *

Reply:
NM looks like you list this below.